src/config.rs

```diff
@@ -34,6 +34,7 @@ const XDS_ADDRESS: &str = "XDS_ADDRESS";
 const CA_ADDRESS: &str = "CA_ADDRESS";
 const TERMINATION_GRACE_PERIOD: &str = "TERMINATION_GRACE_PERIOD";
 const FAKE_CA: &str = "FAKE_CA";
+const CUSTOM_SIGNER: &str = "CUSTOM_SIGNER";
```

Comment:
As we are using a k8s client when custom signer is set, maybe rename as K8S_CUSTOM_SIGNER?

Reply:
sure, will update this one too.